homeassistant/components/media_player/plex.py

```diff
@@ -273,8 +273,23 @@ def __init__(self, config, device, session, plex_sessions,
         self.plex_sessions = plex_sessions
         self.update_devices = update_devices
         self.update_sessions = update_sessions
-
-        self._clear_media()
+        # General
+        self._media_content_id = None
```

Comment:
There's no need to duplicate all of these attributes here as well. Just call `self._clear_media_details()` here. Our linter is smart enough to know it is still defined inside of `__init__`.

Reply:
perhaps you can educate me here. 

Which is best practice? to declare in __init__ or do declare as needed? 
I went with the __init__ declaration as i thought it was more readable and a this why you know which vars are expected. 

Also i'd like to ensure this module conforms to pep8. 

Happy to change per your request but would just like to know which way is preferred and why.